### PR TITLE
KM-39: set max brightness when using library card

### PR DESCRIPTION
**Pull request by aakelaukkanen**
- set app brightness to max when using library card using the `@adrianso/react-native-device-brightness` package
- The package does not update the system/device brightness, only the window brightness. Because of this, the `getBrightnessLevel()` can return max (1) even if the screen is as dark as it can be, because the system brightness (`getSystemBrightnessLevel()`) is overriding the brightness. When `setBrightnessLevel` is called again with the same value, it does not change anything because it reads that the screen is already using that brightness level. This is why the `setBrightnessLevel` must always be called with a different number, either 1 or 0.99 to achieve setting the device brightness correctly to max.
- main app pull request of this: https://github.com/City-of-Kouvola/Library_card_module/pull/28

### CODE REVIEW

File: package.json
Comment (EetuKaunismaki) [inline package.json:1]: Nosta vielä paketin versiota jotta npm tunnistaa sen muuttuneen

Comment (aakelaukkanen) [reply package.json:1]: done

File: src/modules/LibraryCard/components/LibraryCard.tsx [hunk 1]
```diff
@@ -29,6 +31,28 @@ export const LibraryCard = ({cardNumber, holderName, logout}: IProps) => {
     }, 5000);
   }, [])  
 
+  useEffect(() => {
+
+    if (!isFocused) {
+      return;
+    }
+
+    getBrightnessLevel().then(brightness => {
```
[the remark this answers is not in the document]
Comment (EetuKaunismaki) [reply src/modules/LibraryCard/components/LibraryCard.tsx:40]: Ehkä import { NativeModules } from 'react-native'; ja NativeModules.RNDeviceBrightnessModule.getBrightnessLevel() kutsuilla toimisi varmemmin? pitää testata

File: package.json [hunk 2]
```diff
@@ -10,9 +10,10 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "peerDependencies": {
+    "@react-native-async-storage/async-storage": ">=1.19.1",
     "react": ">=18.2.0",
     "react-native": ">=0.72.3",
-    "@react-native-async-storage/async-storage": ">=1.19.1"
+    "@adrianso/react-native-device-brightness": "^1.2.7"
```
Comment (EetuKaunismaki) [inline package.json:16]: PeerDependencyn kanssa kannattaa olla joustavami kuin muualla, esim. tässä vois olla suurempi kyin eikä vain minor versiot

**Review by EetuKaunismaki — APPROVED** [review]

**Pull request merged**